spec/workers/virus_scan_worker_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe VirusScanWorker, type: :worker do
```

Comment:
Very minor: What does `type: :worker` do and do we definitely need it?

Reply:
No - it was added by the sidekiq rspec generator, I'll remove it.